src/components/dashboard/Dashboard.jsx

```diff
@@ -82,11 +74,25 @@ const Dashboard = (props) => {
             {handleSubmit}) => (
             <form onSubmit={handleSubmit}>
```

Comment:
form nested in a form?

Reply:
<form> inside the <Form>. It's a Final-form approach.